portfolio/README.md

```diff
@@ -1,8 +1,10 @@
+![](https://github.com/mmurray/my-portfolio/workflows/Java_CI/badge.svg)
```

Comment:
I don't understand how this is going to work. Will this file be auto-generated? It does not exist right now in this branch.

Reply:
yes this file will be autogenerated :D And it can be seen directly in the Github README.md. I will eventually create a pull request solely dedicated to github workflows so we discuss these techniques further!